From e8c2ded8774f9e12800a1627fb17d0adc7a18632 Mon Sep 17 00:00:00 2001
From: Yelena <yelena@marvell.com>
Date: Mon, 7 Nov 2016 16:10:13 +0200
Subject: [PATCH 1073/2241] defconfig: enable dsa and switch drivers

    These drivers are used to work with external switches (SOHO family).

Change-Id: Ib28cf0784fe7974f9ee5979768537d4e020f193f
Signed-off-by: Yelena <yelena@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33600
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 67f3b5b..85a0d42 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -62,8 +62,10 @@ CONFIG_IP_PNP_DHCP=y
 CONFIG_IP_PNP_BOOTP=y
 # CONFIG_INET_LRO is not set
 CONFIG_BRIDGE=y
+CONFIG_NET_DSA=y
 CONFIG_VLAN_8021Q=y
 CONFIG_OPENVSWITCH=y
+CONFIG_NET_SWITCHDEV=y
 CONFIG_BPF_JIT=y
 # CONFIG_WIRELESS is not set
 CONFIG_NET_9P=y
@@ -112,6 +114,7 @@ CONFIG_MACVTAP=y
 CONFIG_TUN=y
 CONFIG_VIRTIO_NET=y
 CONFIG_VHOST_NET=y
+CONFIG_NET_DSA_MV88E6XXX=y
 CONFIG_E1000=y
 CONFIG_E1000E=y
 CONFIG_IGB=y
-- 
2.7.4

